docs/ORTModule_Training_Guidelines.md

```diff
@@ -495,3 +495,31 @@ for epoch in range(start_epoch, n_epochs):
 ```
 
 Check [LoadBalancingDistributedBatchSampler implementation](../orttraining/orttraining/python/training/utils/data/sampler.py) for more details.
+
+## 8 Using ORTPipelineModule for Deepspeed Pipeline Parallel
+
+You can use `ORTPipelineModule` to support Deepspeed Pipeline Parallelism. Here's how you can integrate it into your pipeline:
+
+```python
+from onnxruntime.training.ortmodule import DebugOptions
```

Comment:
It's great you include the required packages in example. Sometime, it's not easy to find how to import the right components, especially, when they are renamed by `import x as y` somewhere.

Reply:
like I mentioned before, these are the only changes the user should do to switch from deepspeed+pytorch to deepspeed+ort.
we will include a full working example in the training-examples repo

Reply:
Just detailed instructions on how to run your tests should be fine. No need to spend time for another example at this stage.